lib/tests/test_diff_summary.rs

```diff
@@ -13,7 +13,7 @@
 // limitations under the License.
 
 use jj_lib::matchers::{EverythingMatcher, FilesMatcher};
-use jj_lib::merged_tree::{DiffSummary, MergedTree};
+use jj_lib::merged_tree::DiffSummary;
 use jj_lib::repo_path::RepoPath;
 use test_case::test_case;
 use testutils::TestRepo;
```

Comment:
nit: `create_tree()` can be imported by name

Reply:
Done